src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -198,12 +198,11 @@ class DefaultStorageKeyAllocator
     }
     static StorageKeyName SubscriptionResumptionMaxCount() { return StorageKeyName::Formatted("g/sum"); }
 
-    static StorageKeyName GlobalSceneCountKey() { return StorageKeyName::Formatted("g/scc"); }
```

Comment:
Were these removed on purpose?

Reply:
Nope. Must have been a rebase error. That explains why builds on this PR were always failing. I will fix it.